### PR TITLE
adding prim's algo and kadane's algo in c

### LINKED ISSUE #265: Minimum spanning tree algorithms - Prim's and Kruskal's
Minimum spanning tree algorithms - Prim's and Kruskal's

**Describe the solution you'd like**
A minimum spanning tree (MST) or minimum weight spanning tree is a subset of the edges of a connected, edge-weighted un-directed graph that connects all the vertices together, without any cycles and with the minimum possible total edge weight.
There are two greedy algorithms - Prim's and Kruskal's to solve this problem.

**Additional context**
Minimum spanning trees have direct applications in the design of networks, including computer networks, telecommunications networks, transportation networks, water supply networks, and electrical grids.

**Languages**

-   [ ] Python : ?
-   [x] Java : @theDeepanshuMourya
-   [x] C : @rgarg-2711
-   [x] C++: @ManasUniyal

### PR DESCRIPTION
**Pull request by rgarg-2711**
# Description # i am prim's algorithm and implemenation for finding minimum spanning tree

Please include a summary of the changes you have made in regard with this issue. 

Fixes #265

## Type of change

**Please delete options that are not relevant**

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented in my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by rgarg-2711** [comment]
thank you for assigning me.

**Comment by rgarg-2711** [comment]
in the reference to issue #265

File: C/PrimMST.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/* A C program for Prim's Minimum Spanning Tree (MST) algorithm. The program is for adjacency matrix representation of the graph*/
```
Comment (kaustubhgupta) [inline C/PrimMST.c:1]: Update C folder readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: C/PrimMST.c (hunk 1 above)
Comment (rgarg-2711) [reply C/PrimMST.c:1]: ohk sir i will

**Comment by rgarg-2711** [comment]
in reference to issue #265 update c folder readme

**Comment by rgarg-2711** [comment]
in reference to issue #287 adding kadane's algorithm in c.thank you for assign me.

**Comment by YashSaxena9** [comment]
Your choice is exact copy from geeksforgeeks website, your PR is being rejected for violating our contribution guide.
We don't accept plagiarising contributions.

**Review by YashSaxena9 — COMMENTED** [review]
Plagiarism detected

**Pull request closed without merging**